### PR TITLE
Add `markup`

### PR DESCRIPTION
**Pull request by utkarshkukreti**
I released this crate today. The performance seems to be roughly around the fastest implementations (write! / fomat / horrorshow) but I'll let you update the numbers in the README.md whenever you get a chance!

### CODE REVIEW

File: src/markup_bench.rs [hunk 1]
```diff
@@ -0,0 +1,82 @@
+use criterion;
+
+// TODO: Switch to `markup::define!` when upgrading to Rust 2018.
```
Comment (djc) [inline src/markup_bench.rs]: Best get that out of the way now? :)

File: Cargo.toml [hunk 2]
```diff
@@ -10,6 +10,7 @@ criterion = "0.2"
 handlebars = { git = "https://github.com/sunng87/handlebars-rust" }
 horrorshow = { git = "https://github.com/Stebalien/horrorshow-rs" }
 liquid = { git = "https://github.com/cobalt-org/liquid-rust" }
+markup = "0.1"
```
Comment (djc) [inline Cargo.toml]: Please depend on Git master here, as is done for the other crates. This is part of providing a level playing field: all implementations get the performance from their very latest versions.

File: src/markup_bench.rs (hunk 1 above)
Comment (utkarshkukreti) [reply src/markup_bench.rs]: Sorry do you mean remove this comment or upgrade the crate to 2018?

**Comment by utkarshkukreti** [comment]
> Just know that, if things ever break (because of API/syntax changes for example), I might remove your crate rather than fixing up the benchmarks for it -- but I'll give you some time to fix them up.

Sure, no problem!

**Comment by djc** [comment]
Pff, sorry this has taken me so long!

**Pull request merged**